### ISSUE TITLE
support multi-device screenshot for one dataset

### ISSUE THREAD

**Issue by satra** [dandi/dandi-api-webshots#13]
it would indeed be nice tosee how things look on different devices (phone, ipad, computer). not urgent, but would be a nice set of things to generate. this multi-device view could be done just for the test dataset for example, instead of every single dataset.

**Comment by yarikoptic** [comment]
good idea.  What dandiset would be your favorite for this purpose?  we might then want also to add traversal of the tree into subdirectory and back (https://github.com/dandi/dandiarchive/issues/600)
